src/main/java/com/bravo/user/model/dto/AddressReadDto.java

```diff
@@ -0,0 +1,19 @@
+package com.bravo.user.model.dto;
+
+import lombok.Data;
+
+import java.time.LocalDateTime;
+
+@Data
+public class AddressReadDto {
+
+    private String id;
+    private String address;
+    private String line1;
+    private String line2;
```

Comment:
If a reviewer sees line1 and line2, they may not know what this is.  Even inside of an address-related object.  I've seen all the way up to 5 of these in codebases that handle international addresses :) .